mobile/src/modules/auth/screens/AccountsScreen.tsx

```diff
@@ -82,6 +82,10 @@ class Accounts extends React.PureComponent<TProps, State> {
     this.setModalVisible(false);
   }
 
+  handleDeleteAccount = (_account: Account) => {
+    // TODO: do something;
+  }
```

Comment:
FYI account deletion is already possible through swiping an account on the HomeScreen, see https://github.com/burst-apps-team/phoenix/blob/develop/mobile/src/modules/auth/screens/HomeScreen.tsx#L97